packages/labs/ssr-dom-shim/src/lib/events.ts

```diff
@@ -0,0 +1,567 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * This implementation is not fully spec compliant (e.g. validation),
+ * but should work well enough for our use cases.
+ */
+
+type EventTargetInterface = EventTarget;
+
+const isCaptureEventListener = (
+  options: undefined | AddEventListenerOptions | boolean
+) => (typeof options === 'boolean' ? options : options?.capture ?? false);
+
+// Shim the global EventTarget object
+const EventTargetShim = class EventTarget implements EventTargetInterface {
```

Comment:
I think we definitely need some tests for this.

Reply:
This is indirectly and directly tested in the ssr package. Should I add additional tests in the shim package?